pkgs/applications/science/biology/obitools/obitools3.nix

```diff
@@ -13,15 +9,20 @@ pythonPackages.buildPythonApplication rec {
     sha256 = "1x7a0nrr9agg1pfgq8i1j8r1p6c0jpyxsv196ylix1dd2iivmas1";
   };
 
+  disabled = python3Packages.pythonOlder "3.5";
+
```

Comment:
This can be safely removed. python3 points to 3.9 right now.

Reply:
Will create a new PR for that

Reply:
https://github.com/NixOS/nixpkgs/pull/145034